README.md

```diff
@@ -91,6 +91,11 @@ export interface IResultOptions {
    * @default true
    */
   resolveOnVersionControl?: boolean;
+  /**
+   * @description The version of the npm package (when `resolveOnNpmRegistry` only) to retrieve the scorecard for.
+   * @default "latest"
+   */
+  version?: string;
```

Comment:
version feel like a bit too simplistic no? Like version of what?

`npmPackumentVersion` or something like that instead?

Reply:
Thought the doc + jsdoc would be enough to make it clear but I agree `version` is not explicit.
`packageVersion` ?
or `npmPackageVersion` if we wanna be explicit that this option is for npm only.
`npmPackumentVersion` could represents the version of packument itself, no ?

Reply:
npmPackageVersion is fine

Reply:
Done